testing/busybench.js

```diff
@@ -15,16 +15,18 @@
  */
 
 const startTime = Date.now();
+const test_arr = [];
 
 /**
- * Repeatedly fills a buffer, then calls itself with setImmediate.
+ * Fills several arrays, then calls itself with setImmediate.
  * It continues to do this until durationSeconds after the startTime.
  */
 function benchmark(durationSeconds) {
-  var buffer = new Buffer(1e4);
-  for (var k = 0; k < 1e4; k++) {
-    for (var j = 0; j < buffer.length; j++) {
-      buffer[j] = 0;
+  //_.test_buffer = new Buffer(1e8);
+  for (let i = 0; i < 10; i++) {
+    test_arr[i] = new Array(1e5);
```

Comment:
The allocation can be done once, doesn't need to be done on every benchmark() call.

Reply:
Okay -- only make new arrays on the first call. 
I'm still storing the result of the computation in the array, to make sure it isn't optimized away.